nightwatch.config.js

```diff
@@ -0,0 +1,34 @@
+require('babel-register')
```

Comment:
How much of this is specific to how circle CI needs to be setup?

Reply:
it's weird because gymnast doesn't run well locally. You first need to deploy to CI and then you can run with `URL=whereveritwasdeployed yarn test:image`.

That being said, if you specify URL this also works locally (assuming storybook was deployed elsewhere)